pretty_midi/instrument.py

```diff
@@ -34,6 +34,8 @@ class Instrument(object):
         The program number of this instrument.
     is_drum : bool
         Is the instrument a drum instrument (channel 9)?
+    is_vocals : bool
+        Is the 'instrument' a vocalist, synchronized with the lyrics
```

Comment:
It wouldn't be synchronized with the lyrics if self.is_vocals is set manually; I would just say "Is the instrument a vocalist?"

Reply:
Wrong choice of words, I was referring to the procedure, noted.